_stbt/pylint_plugin.py

```diff
@@ -35,8 +29,19 @@ class StbtChecker(BaseChecker):
     msgs = {
         # Range 70xx reserved for custom checkers: www.logilab.org/ticket/68057
         'E7001': ('Image "%s" not found on disk',
+                  'stbt-missing-image',
                   'Used when the image path given to `stbt.wait_for_match` '
                   '(and similar functions) does not exist on disk.'),
+        'E7002': ('"%s" return value not used (missing "assert"?)',
+                  'stbt-unused-return-value',
+                  "When the return value from stbt's "
+                  "wait_until/match/is_screen_black isn't used in an 'if' "
```

Comment:
This list of functions doesn't match up with the ones used in the regex below (missing `match_text` and `ocr`), and I'm not sure of the value of listing every function that we check for here.  Perhaps '"%s" does not throw on failure and the return value is not used. Perhaps you meant "assert %s(...)"'.